spec/thor/zsh_completion/generator_spec.rb

```diff
@@ -8,6 +8,13 @@ def baz
         p options
         puts "baz"
       end
+      map 'b' => 'baz'
+
+      desc "foo-bar", "Dashed command"
```

Comment:
Thanks.

>       foo-bar)
>           __generator_spec_nest1_nest2_foo-bar

Does not this completion script run correctly?

In my environment, zsh completes `foo-bar` subcommand after following steps.

```
# zsh 5.2 (x86_64-apple-darwin16.0)
# Save https://gist.github.com/labocho/12ead330f5112d06c6f5a2fa4fa6bc5d to generator_spec.zsh
$ source generator_spec.zsh
$ generator_spec nest1 nest2 [TAB] # shows `foo-bar` etc.
```